src/components/PopoverWithMeasuredContent.js

```diff
@@ -59,138 +59,115 @@ const defaultProps = {
  * This way, we can shift the position of popover so that the content is anchored where we want it relative to the
  * anchor position.
  */
-class PopoverWithMeasuredContent extends Component {
-    constructor(props) {
-        super(props);
-
-        this.popoverWidth = lodashGet(this.props, 'popoverDimensions.width', 0);
-        this.popoverHeight = lodashGet(this.props, 'popoverDimensions.height', 0);
 
-        this.state = {
-            isContentMeasured: this.popoverWidth > 0 && this.popoverHeight > 0,
-            isVisible: false,
-        };
-
-        this.measurePopover = this.measurePopover.bind(this);
-    }
+function PopoverWithMeasuredContent(props) {
+    const {windowWidth, windowHeight} = useWindowDimensions();
+    const [popoverWidth, setPopoverWidth] = useState(props.popoverDimensions.width);
+    const [popoverHeight, setPopoverHeight] = useState(props.popoverDimensions.height);
```

Comment:
`props.popoverDimensions` is not always defined - see [this usage](https://github.com/Expensify/App/blob/7b9afbe12478cb9f5c7560451b01c07ea9c9c239/src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js#L294-L305). We should add a default as trying to access `.width` of undefined will throw